### ISSUE TITLE
Character Idea Brain Storm

### ISSUE THREAD

**Issue by Psyceric** [connormarry/Smash-like-game#2]
We need you to list any Character ideas that you make, if you have an idea please elaborate with how this could be interesting to the player, and not just an idea. A short snippet will be needed for anyone who is doing concept art if they enjoy the idea, and make something of it.

Please leave any ideas in the area below, and we will discus them

**Comment by connormarry** [comment]
Rah- speedy approaches and strong punish game. No projectiles and good recovery. Blue hair/ Red skin

Olimo- Slow, but very strong approaches. Weak on the neutral but great on grab conversions. Good Recovery and uses projectiles (corn cobs) Wears a tank top.

**Comment by connormarry** [comment]
Thrall- Swordsman. Tip of sword gives unorthodox knockback (mainly straight up and low knockback depending on the move) which helps combo game. Part of sword close to the hand gives strong orthodox knockback and secures kills. Middle gives normal orthodox knockback and is the least effective hitbox of the sword.  Normal weight. moderate neutral game. strong punish game/ combo game. Has a hunchback and wears a red cape.

**Comment by staltaro** [comment]
Should they all have at least one special ability? Something fair but gives diversity to each of the characters.

**Comment by Psyceric** [comment]
I feel they should all have a ability to use a ultimate, similar to how they can in other fighting games. This could add a skill gap for more advanced players to be able to know when it's happening ect

The ability shouldn't be broken though, and we should have it be able to be countered if the person on the other end is skilled enough

**Comment by connormarry** [comment]
Yeah well the smash competitive scene doesnt use ults. it is more combo oriented. each character will have a unique set of moves for the b and a buttons relative to the position of the character and the direction of the control stick and/or c stick.
